atom/renderer/api/atom_api_spell_check_client.cc

```diff
@@ -155,62 +148,93 @@ void SpellCheckClient::SpellCheckText(
   base::string16 word;
   int word_start;
   int word_length;
+  std::vector<base::string16> words;
+  auto& word_map = pending_request_param_->wordmap();
   for (auto status =
            text_iterator_.GetNextWord(&word, &word_start, &word_length);
        status != SpellcheckWordIterator::IS_END_OF_TEXT;
        status = text_iterator_.GetNextWord(&word, &word_start, &word_length)) {
```

Comment:
_(opinion)_

Those two calls to GetNextWord() in the loop structure are kind of cumbersome. This would be less repetitive:
```cc
for (;;) {
  const auto status = text_iterator_.GetNextWord(...);
  if (status == SpellcheckWordIterator::IS_END_OF_TEXT)
    break;
  if (status == SpellcheckWordIterator::IS_SKIPPABLE)
    continue;
```